bsdlib/README.rst

```diff
@@ -13,111 +13,30 @@ The library has the following set of sockets:
 * 1 modem DFU socket
 * 10 PDN sockets
 
-Overview
-********
+The BSD library is the primary interface for operating the nRF9160 modem and to establish LTE-M, NB-IoT, and GNSS connections.
+The library abstracts away the details on memory management and synchronization, allowing you to focus on writing the application using standardized API calls.
 
 .. figure:: doc/images/bsd_lib_overview.svg
    :alt: BSD library overview diagram
 
    BSD library overview diagram
 
-The BSD library is the primary interface to operate the nRF9160 modem in order to establish LTE-M, NBIOT, and GNSS connections.
-The library abstracts away the details on memory management and synchronization allowing you to focus on writing the application using standardized API calls.
+Socket programming is difficult.
```

Comment:
This is a strange line to include in the documentation.

Reply:
But it is! ;) 
But okay, reworded ...